### ISSUE TITLE
预测测试集样本所属的类别，经常发生改变？

### ISSUE THREAD

**Issue by jerryhero** [PanJinquan/tensorflow_models_nets#5]
用图片训练好inception_v3模型后，再用其他图片做预测，为什么对于测试集样本，每次预测所属的类别经常变动？

**Comment by ghost** [comment]
我也遇到了同样的问题，至今还没发现是哪里的问题，改一下batch的大小的话，预测出来的每个类别的概率也会发生变化

**Comment by Instantnoodles-madman** [comment]
Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

2 root error(s) found.
  (0) Not found: Key InceptionV3/AuxLogits/Conv2d_1b_1x1/BatchNorm/beta not found in checkpoint
	 [[node save/RestoreV2 (defined at predict.py:31) ]]
这种问题怎么解决啊  请教一下
